.vscode/settings.json

```diff
@@ -10,9 +10,9 @@
   "files.exclude": {
     "**/.DS_Store": true,
     "**/.git": true,
-    "**/node_modules": true,
-    "**/out": true,
-    "**/dist": true,
+    "**/node_modules": false,
+    "**/out": false,
+    "**/dist": false,
```

Comment:
Do we need to update this, or can this be kept as local vscode override? Did you need to access these folders (they should never be tweaked as they are products of build).

Reply:
added .vscode folder to gitignore since it's not supposed to change 🙏